config/operator/namespace/cluster_role_binding.template.yaml

```diff
@@ -0,0 +1,14 @@
+# allow operator to manage resources in its namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: elastic-operator
+  namespace: <NAMESPACE>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: elastic-operator
+subjects:
+- kind: ServiceAccount
+  name: elastic-namespace-operator
+  namespace: <NAMESPACE>
```

Comment:
The file name does not seem to be aligned with its contents (RoleBinding != cluster_role_binding).